docs/themes/porter/assets/sass/helm-blog.scss

```diff
@@ -60,6 +7,7 @@
       .content-wrap {
         h1 {
           margin-top: 0;
+          color: $gray;
```

Comment:
I can't tell if this change was intentional? I'm seeing the listing page for the blog give each blog title entry a background color which looks a bit off to me.

![Screen Shot 2021-05-17 at 3 50 41 PM](https://user-images.githubusercontent.com/1368985/118554967-bd272980-b727-11eb-8be1-eb2db3280474.png)

Reply:
I've reverted these color changes back. I'd like to introduce more of the Porter colors from the branding last year, but that's better addressed separately I think to tweak the theme overall. I'll stick to the bug fixes for now

<img width="495" alt="Screen Shot 2021-05-17 at 1 56 17 PM" src="https://user-images.githubusercontent.com/686194/118557426-04a6b900-b71a-11eb-9fae-18a2d7aacca9.png">

Reply:
I love the purple, noticed that immediately on the dates. So looking forward to the next PR. 👍